tests/Stubs/LoginForm.php

```diff
@@ -86,4 +87,14 @@ private function loginRules(): array
             new Email()
         ];
     }
+
+    private function passwordRules(): array
+    {
+        return [
+            new Required(),
+            (new HasLength())
```

Comment:
This rule should be removed. It's a bad practice to limit password length.

Reply:
```
        return [
            new Required(),
            (new HasLength())
            ->min(8)
            ->tooShortMessage('Is too short.'),
        ];
```

Reply:
done.